src/main/flight/pid.c

```diff
@@ -547,43 +556,52 @@ rollAndPitchTrims_t *angleTrim, rxConfig_t *rxConfig)
 #endif
 
 #ifdef AUTOTUNE
-            if (shouldAutotune()) {
+            if (shouldAutotune())
+            {
                 error = DEGREES_TO_DECIDEGREES(autotune(rcAliasToAngleIndexMap[axis], &inclination, DECIDEGREES_TO_DEGREES(error)));
             }
 #endif
             PTermACC = error * (float)pidProfile->P8[PIDLEVEL] * 0.008f;
             tmp0flt = (float)pidProfile->D8[PIDLEVEL] * 5.0f;
```

Comment:
@mj666: gcc is clever enough to generate good code. Local block definition is actually easier to optimize, compiler does not have to infer that variable value is unused in rest of code.